contracts/contracts/coordination/GlobalAllowList.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.8.0;
+import "@openzeppelin/contracts/access/AccessControlDefaultAdminRules.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "./IRitualAuthorizer.sol";
+import "./Coordinator.sol";
+
+contract GlobalAllowList is AccessControlDefaultAdminRules, IRitualAuthorizer {
+    using ECDSA for bytes32;
+
+    Coordinator public coordinator;
```

Comment:
```suggestion
    Coordinator public immutable coordinator;
```

Reply:
Can this variable be immutable - there is a `setCoordinator(...)` function?

Reply:
you're right, thanks